modules/tinymce/src/core/main/ts/fmt/FormatUtils.ts

```diff
@@ -121,12 +122,8 @@ const getNonWhiteSpaceSibling = (node: Node | null, next?: boolean, inc?: boolea
   return undefined;
 };
 
-const isTextBlock = (schema: Schema, name: string | Node): boolean => {
-  if (isNode(name)) {
-    name = name.nodeName;
-  }
-
-  return !!schema.getTextBlockElements()[name.toLowerCase()];
+const isTextBlock = (schema: Schema, node: Node): boolean => {
+  return !!schema.getTextBlockElements()[node.nodeName.toLowerCase()] || TransparentElements.isTransparentBlock(schema, node);
 };
```

Comment:
```suggestion
const isTextBlock = (schema: Schema, node: Node): boolean =>
  !!schema.getTextBlockElements()[node.nodeName.toLowerCase()] || TransparentElements.isTransparentBlock(schema, node);
```
This can be trimmed down now.